yeti.conf.sample

```diff
@@ -1,3 +1,20 @@
+[auth]
+
+##
+## Use these settings to configure Yeti authentication.
+## If you do not specify a username and password, there will be no authentication.
```

Comment:
What do you mean by `If you do not specify a username and password, there will be no authentication.`?

Reply:
Erf... Wrong copy & paste. Should be better now.